lib/states/wi/madison.R

```diff
@@ -8,86 +8,137 @@ source("common.R")
 # each year, this seems reasonable.
 load_raw <- function(raw_data_dir, n_max) {
   # NOTE: "IBM" is the officers department ID
-  cit <- load_single_file(
+  cit_old <- load_single_file(
```

Comment:
Might be good to add a short comment distinguishing between the old and new data (e.g., what years they covered, important differences, etc.)